src/sentry/snuba/json_schemas.py

```diff
@@ -16,15 +16,14 @@
             "subscription_id": {"type": "string", "minLength": 1},
             "values": {
                 "type": "object",
-                "minProperties": 1,
-                "additionalProperties": {"type": "number"},
+                "properties": {
+                    "data": {"minProperties": 1, "additionalProperties": {"type": "number"}}
+                },
+                "required": ["data"],
```

Comment:
Didn't set `'additionalProperties': False` here on purpose, so that we're not tied to specifics of the query debug info.

Reply:
This makes sense, I don't even think that we have a clear and consistent schema on our side at this point.